src/main/java/org/example/algorithms/naive.java

```diff
@@ -0,0 +1,28 @@
+public class NaiveSearch {
```

Comment:
- Rename file _naive.java_ to _NaiveSearch.java_
- Add package info at top of class

See other classes for reference.